src/dsp_tools/utils/excel_to_json_properties.py

```diff
@@ -1,14 +1,25 @@
 import importlib.resources
 import json
 import re
+import warnings
 from typing import Any, Optional
+from unittest import mock
 
 import jsonschema
 import pandas as pd
 
 from dsp_tools.models.helpers import BaseError
 from dsp_tools.utils.shared import prepare_dataframe, check_notna
 
+# Pandas relies on openpyxl to parse XLSX files.
+# A strange behaviour of openpyxl prevents pandas from opening files with some formatting properties
+# (unclear which formatting properties exactly).
+# Apparently, the excel2json template files have one of the unsupported formatting properties.
+# The following two lines of code help out.
+# Credits: https://stackoverflow.com/a/70537454/14414188
+p = mock.patch('openpyxl.styles.fonts.Font.family.max', new=100)
+p.start()
+
```

Comment:
This is obscure, but the only way to make it work. If you have a better solution, I'm open for feedback!

Reply:
Would it be possible to find out which of the formatting properties cause the problem and get rid of it?

Reply:
Unfortunately, the Excel file is not exotic. I just took the existing template, and added comments to it. Nothing special. 
After reading the SO answers, I tried to reduce the number of used fonts, but to no avail: The error still appears. It's odd that in some SO answers, the comment boxes seemed to be the root of the problem, but others reported that the number of fonts was the problem.
I think it's a strange behavior of openpyxl that I should perhaps report.

One important reason why I would like to leave the monkey-patching in place: I think it's very probable that the error will also occur on client side.

Reply:
with what program did you do that? some of the comments mentioned WPS causing this issue.

In any case, you should definitely report this problem, if you feel it's a bug in the library.

Reply:
well... ok then. In that case it's probably really the fault of openpyxl